sdks/go/pkg/beam/core/runtime/exec/coder_test.go

```diff
@@ -64,6 +68,18 @@ func TestCoders(t *testing.T) {
 					coder.NewDouble(),
 					coder.NewBool()})}),
 			val: &FullValue{Elm: int64(42), Elm2: &FullValue{Elm: float64(3.14), Elm2: true}},
+		}, {
+			coder: &coder.Coder{Kind: coder.Window, Window: coder.NewGlobalWindow()},
+			val:   &FullValue{Windows: []typex.Window{window.GlobalWindow{}}},
+		}, {
+			coder: &coder.Coder{Kind: coder.Window, Window: coder.NewIntervalWindow()},
+			val:   &FullValue{Windows: []typex.Window{window.IntervalWindow{Start: 0, End: 100}}},
+		}, {
+			coder: coder.NewW(coder.NewVarInt(), coder.NewGlobalWindow()),
+			val:   &FullValue{Elm: int64(13), Windows: []typex.Window{window.GlobalWindow{}}},
+		}, {
+			coder: coder.NewPW(coder.NewString(), coder.NewGlobalWindow()),
```

Comment:
This case could also be split out into its own test to demonstrate the "window in, no window out" expectation